db/migrate/20160923053347_remove_nill_false_from_invitations.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveNillFalseFromInvitations < ActiveRecord::Migration[5.0]
+  def up
+    change_column :invitations, :invited_by_id, :integer,
```

Comment:
@dissident why do you change it? I believe that we always have a user how make invitation